src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -582,6 +582,10 @@ def build_container_from_source_with_acr_task(self, image_name, source):
         from azure.cli.command_modules.acr._client_factory import cf_acr_tasks, cf_acr_runs
         from azure.cli.core.profiles import ResourceType
 
+        # Validate that the source provided is a directory, and not a file.
```

Comment:
In the preview code, the `source` value is infer from `artifact`, the customer not input `--source`, but can see this error when `docker` is not running .